docs/source/v1/BGC/simulation_data/simulation_table.rst

```diff
@@ -0,0 +1,67 @@
+**********************************
```

Comment:
This file would be best reviewed by going to https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_64_try5/html/v1/BGC/simulation_data/simulation_table.html